FinancialTracker/Firebase.swift

```diff
@@ -237,6 +240,18 @@ class FirebaseHandler {
         }
     }
     
+    func addScoreToUserBasedOnTime(_ time: Double, completionHandler: @escaping (FirebaseError?, Bool) -> Void) {
+        guard let uid = self.auth.currentUser?.uid else {
+            completionHandler(FirebaseError.access("Current user is nill in #function."), false)
+            return
+        }
+        
+        let score = ((time / 60) / 20).round(to: 3)
```

Comment:
I need some comments on how the score is calculated.

Reply:
i will add a comment. It is 1 point for every 20 minutes

Reply:
Ok, add and merge.